src/core/unicode.js

```diff
@@ -1630,8 +1630,41 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function normalizeUnicode(str, normalizedUnicodesMap) {
+  if (str.length === 1) {
+    if (normalizedUnicodesMap[str] !== undefined) {
+      return normalizedUnicodesMap[str];
+    }
+    return str;
+  }
+
+  const unicodeBuf = [];
+  for (let i = 0, ii = str.length; i < ii; i++) {
+    const unicode = str[i];
+    const normalized = normalizedUnicodesMap[unicode];
+    if (normalized === undefined) {
+      // If a toUnicode entry contains two same unified Unicode
+      if (unicodeBuf.includes(unicode) === false) {
+        unicodeBuf.push(unicode);
+      }
+      continue;
+    }
+    // If a not unified and *simple* Unicode
+    if (normalized.length === 1) {
+      // If a toUnicode entry contains two same not unified Unicode
+      if (str.includes(normalized) === true ||
```

Comment:
You can simplify this to `if (str.includes(normalized) || unicodeBuf.includes(normalized)) {` so it probably even fits on one line. There is no need to check for `=== true` since it's already a boolean condition.

Reply:
If the condition above isn't correct, and causes regressions, then I really cannot see why this check is generally OK either.